### ISSUE TITLE
Add to readme which folder may be used for single node use

### ISSUE THREAD

**Issue by ShepardEG** [Graylog2/docker-compose#66]
Hello. Not clear which compose file shoud be use for minimal usage.

**Comment by svennd** [comment]
I agree, it would be a good addition, but to anyone looking for a "drive-by" test, I tried open-core/ and that was beginner friendly enough :)
